quickstart.js

```diff
@@ -25,16 +22,11 @@ var client, application, account, group = null;
 var accountEmail = unique('jlpicard') + '@mailinator.com';
 
 // ==================================================
-// Step 1 - Load the API Key file and create a Client
+// Step 1 - Create a client and wait for it to ready
```

Comment:
Yeah, but if you look at the comments they must adhere to the length of the '===='. I think it's because of doc reasons.